Trippy/Utils/ParseUtils.m

```diff
@@ -88,4 +129,22 @@ + (PFObject *)oldPFObjFromLocation:(Location *)loc {
     return [query getFirstObject];
 }
 
++ (PFObject *)oldPFObjFromCollection:(LocationCollection *)col {
+    PFQuery *query = [PFQuery queryWithClassName:@"Collection"];
+    [query whereKey:@"objectId" equalTo:col.parseObjectId];
+    return [query getFirstObject];
+}
+
++ (PFObject *)newPFObjFromItinerary:(Itinerary *)it {
+    PFObject *obj = [PFObject objectWithClassName:@"Itinerary"];
+    obj[@"name"] = it.name;
+    obj[@"createdBy"] = [PFUser currentUser];
+    obj[@"origin"] = [self oldPFObjFromLocation:it.originLocation];
+    obj[@"sourceCollection"] = [self oldPFObjFromCollection:it.sourceCollection];
```

Comment:
If this is the only use of this method, you can remove it from the header entirely.